src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -162,18 +164,24 @@ protected void createTestData() {
         LNPTestData testData = getTestData();
         try {
             createJsonDataFile(testData);
+            persistTestData();
             createCsvConfigDataFile(testData);
-        } catch (IOException ex) {
+        } catch (IOException | HttpRequestFailedException ex) {
             log.severe(TeammatesException.toStringWithStackTrace(ex));
         }
     }
 
     /**
      * Creates the entities in the datastore from the JSON data file.
      */
-    protected void persistTestData() {
+    protected void persistTestData() throws IOException, HttpRequestFailedException {
         DataBundle dataBundle = loadDataBundle(getJsonDataPath());
-        BackDoor.removeAndRestoreDataBundle(dataBundle);
+        String responseBody = "";
+        responseBody = BackDoor.removeAndRestoreDataBundle(dataBundle);
+
+        String pathToResultFile = createFileAndDirectory(TestProperties.LNP_TEST_DATA_FOLDER, getJsonDataPath());
+        String jsonValue = JsonUtils.parse(responseBody).getAsJsonObject().get("message").getAsString();
```

Comment:
I think there is a way such that this (usage of magic string `message`) can be mitigated. Let me get back to you later.